dashboard/analysis_options.yaml

```diff
@@ -29,68 +29,4 @@ include: package:flutter_lints/flutter.yaml
 
 linter:
```

Comment:
Can we remove the linter block here since it's empty?

Reply:
for sure